src/steamship/invocable/entrypoint.py

```diff
@@ -0,0 +1,19 @@
+"""
+This class is necessary to be able to please the entrypoints of both localstack and AWS.
+
+If we set the entrypoint directly to steamship.invocable.safe_handler (imported in the init
```

Comment:
Wow! Reads like it has been a struggle to figure things out. Thanks for putting in the effort!

Reply:
Yeah, that bit was painful, and wanted the reasoning to live on for posterity, so that we didn't forget about it later and mistakenly see it as an opportunity for cleanup.